app/assets/stylesheets/application.css.scss

```diff
@@ -35,14 +35,32 @@
   height: 100vh;
   position: relative;
 }
+ // h1{
+ //   font-family: 'Poiret One', cursive;
+ // }
 
+.trophy-case-container {
+  background-image: url('http://urlm.co/blog/wp-content/uploads/ipad23.jpg');
```

Comment:
Prefer double-quoted strings<br>URL `http://urlm.co/blog/wp-content/uploads/ipad23.jpg` should not contain protocol or domain